### PR TITLE
Make configuration more versatile

### PR DESCRIPTION
**Pull request by schu**

### CODE REVIEW

File: builder-config [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# *** Don't touch this file, instead use .config or environment variables ***
+
+# Working directory
+readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# The kernel version to build and include
+S1B_KERNEL_VERSION="${S1B_KERNEL_VERSION:-"4.9.5"}"
```
Comment (alban) [inline builder-config:8]: What does "S1B_" mean?

Comment (schu) [reply builder-config:8]: **S**tage**1B**uilder :)

**Comment by alban** [comment]
Except my question above, LGTM

File: builder-config (hunk 1 above)
Comment (alban) [reply builder-config:8]: ah ok. I was reading https://en.wikipedia.org/wiki/S1B_reactor

**Pull request merged**